readthedocs/doc_builder/config.py

```diff
@@ -26,16 +27,26 @@ def load_yaml_config(version):
 
     img_name = project.container_image or DOCKER_IMAGE
     python_version = 3 if project.python_interpreter == 'python3' else 2
+    allow_v2 = project.has_feature(Feature.ALLOW_V2_CONFIG_FILE)
```

Comment:
Not sure if this is the best way of using the feature flag here.

Reply:
Let me know if this is okay, to generate the migration.

Reply:
I wanted to add this to don't allow users to use the v2 in production yet. Maybe a django setting is the best here? or what name do you suggest for this feature flag?